java/core/src/main/java/co/worklytics/psoxy/rules/slack/PrebuiltSanitizerRules.java

```diff
@@ -0,0 +1,65 @@
+package co.worklytics.psoxy.rules.slack;
+
+import co.worklytics.psoxy.Rules;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+import static co.worklytics.psoxy.Rules.Rule;
+
+/**
+ * Prebuilt sanitization rules for Slack Discovery API responses
+ */
+public class PrebuiltSanitizerRules {
+
+    static final Rules SLACK = Rules.builder()
+        .allowedEndpointRegex("^/api/discovery.conversations.list.*")
```

Comment:
should `.` here be escaped, except for last? as in a regex it's all characters - so I assume not what you mean?

Reply:
good catch